libiocage/lib/JailConfig.py

```diff
@@ -213,7 +213,7 @@ def _set_name(self, name, **kwargs):
             self["id"] = name
         else:
             try:
-                self["id"] = str(uuid.UUID(name)) # legacy support
+                self["id"] = str(uuid.UUID(name))  # legacy support
```

Comment:
Since this is for legacy support, I think `uuid` would be a better indicator of what the id is. As id can be a bit ambiguous

Reply:
The `ID` brings together all the concepts of name and uuid :) It's the only true identifier of a jail.